eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -167,6 +167,34 @@
         }
       }
     },
+    {
+      "StaticConfigs": {
+        "Ubuntu22": {
+          "OSVmImage": "MMSUbuntu22.04",
+          "Pool": "azsdk-pool-mms-ubuntu-2204-general",
+          "VCPKG_DEFAULT_TRIPLET": "x64-linux",
+          "BuildArgs": "-j 10",
+          "RunProxyTests": true
+        }
+      },
+      "BuildSettings": {
+        "clang-13": {
```

Comment:
Can you confirm that this step is failing in CI before your fix?

Reply:
I can confirm that this step is NOT failing in CI before my fix.

The customer reported problem requires a particular version of cstdlib but I don't know where their version of cstdlib came from.